### ISSUE TITLE
invalid version detection in install_command.ps1 

### ISSUE THREAD

**Issue by hh** [chef/mixlib-install#16]
As a chef-provisioning user
When I set chef_version in convergence_options to 12.5.1
Then I expect a node with 12.4.1 installed to upgraded to 12.5.1.

(If this needs to be a chef-provisioning ticket, let me know)

Suspecting something around https://github.com/chef/mixlib-install/blob/14611366684808e4db54dfaeee8ca49a8816a4e6/support/install_command.ps1#L12

```
               convergence_options: {
                 chef_version: '12.5.1'
               },
```

```
    - write file C:\chef\client.rb on base-2012-hardened-xviii
    - write file C:\chef\install.ps1 on base-2012-hardened-xviii
    [base-2012-hardened-xviii] Method invocation failed because [Selected.System.String] does not contain a 
                               method named 'StartsWith'.
    [base-2012-hardened-xviii] -----> Chef Omnibus installation detected (12.5.1)
                               
    [base-2012-hardened-xviii] At C:\chef\install.ps1:19 char:7
                               +   if ($chef_version.StartsWith($version)) { return $false }
                               +       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                   + CategoryInfo          : InvalidOperation: (StartsWith:String) [], Runtim 
                                  eException
                                   + FullyQualifiedErrorId : MethodNotFound
                                
    - run 'powershell.exe -ExecutionPolicy Unrestricted -NoProfile "& ""C:\chef\install.ps1"""' on base-2012-hardened-xviii[2015-10-14T09:46:01-04:00] INFO: Processing chef_node[base-2012-hardened-xviii] action create (basic_chef_client::block line 57)
[2015-10-14T09:46:02-04:00] INFO: Converging base-2012-hardened-xviii because the resource was updated ...

    [base-2012-hardened-xviii] [2015-10-14T13:46:18+00:00] INFO: *** Chef 12.4.1 ***
```

**Comment by thommay** [comment]
@jaym @smurawski can y'all have a look at this? I lifted the powershell code from kitchen I think.

**Comment by jaym** [comment]
maybe related to https://github.com/test-kitchen/test-kitchen/pull/864

**Comment by dissonanz** [comment]
Yes it's the same. I had to refactor the split in order to stay within the command size limit, but this seems to work.

**Comment by schisamo** [comment]
Fix is in https://github.com/chef/mixlib-install/pull/21

**Comment by patrick-wright** [comment]
Fixed in mixlib-install 0.7.1
